### ISSUE TITLE
Line breaks in pattern parameters

### ISSUE THREAD

**Issue by dennisfrank** [pattern-lab/patternlab-node#473]
I am using Pattern Lab Node `v2.5.1` on `Mac`, with Node `v5.3.0`, using a custom `Grunt` Edition.

Coming from Pattern Lab PHP I often format more complex pattern parameters with line breaks for better clarity. This seems to break the pattern include and nothing gets included at all:

```
{{> patternType-pattern(
    attribute1: value,
    attribute2: "value string",
    attribute3: "value string"
    ) }}
```

**Comment by uxmfdesign** [comment]
Having the same issue.

**Comment by bmuenzenmeyer** [comment]
I'd start by playing with the regular expressions in each engine to recognize the new lines, if needed. Example: https://github.com/pattern-lab/patternengine-node-handlebars/blob/master/lib/engine_handlebars.js#L28-L30

they are used here: https://github.com/pattern-lab/patternlab-node/blob/master/core/lib/object_factory.js#L133

See what happens then. If things are still not working as expected after that, a second step may be needed on the parse side

**Comment by ajilderda** [comment]
This would be a great improvement. I've also found that using `()` inside a pattern parameter causes the pattern to not receive any data. For example this does not work:
`{{> molecules-pattern( title: "this is causing the issue ()") }}`
This does work:
`{{> molecules-pattern( title: "this is causing the issue") }}`
Maybe this related or would you like to add an issue for this?

**Comment by bmuenzenmeyer** [comment]
this is likely a regex problem, and is a separate issue. thanks for the report!

**Comment by raphaelokon** [comment]
@bmuenzenmeyer Do you want me to fix this?

**Comment by bmuenzenmeyer** [comment]
@raphaelokon if you are referring to @ajilderda's [comment](https://github.com/pattern-lab/patternlab-node/issues/473#issuecomment-318584549), sure thing!

**Comment by JosefBredereck** [comment]
Is this a related topic to mustache which is using a custom pattern reference?

Pattern
```html
<p>{{text}}</p>
<p>{{text2}}</p>
```

Reference
```
{{> 
    pattern
    text="Hello"
    text2="Hello2"
}}
```

With handlebars, it is working right now. I don't know when the support for mustache will be dropped, so maybe still relevant.

**Comment by bmuenzenmeyer** [comment]
this is a won't fix in the same vein as #145 - if the template language doesnt support it, we wont be doing it.
